src/userbase-js/types/index.d.ts

```diff
@@ -0,0 +1,68 @@
+/* auth */
+
+export interface Session {
+  sessionId: string
+  creationDate: string
+}
+
+export interface UserProfile {
+  [key: string]: string
+}
+
+export interface UserResult {
+  username: string
+  key: string
+  email?: string
+  profile?: UserProfile
+}
+
+export type ShowKeyHandler = (seedString: string, rememberMe: boolean, backUpKey: string) => void | Promise<void>
+
+export function init(options: { appId: string, endpoint?: string, keyNotFoundHandler?: () => void }): Promise<Session>
```

Comment:
Assuming we'd want an explicit type for the `keyNotFoundHandler` same as with the `ShowKeyHandler`

It's a synchronous function that takes in the following as params: `(username: string, deviceId: string)` ([source](https://github.com/encrypted-dev/userbase/blob/172ce59f6c69b7d5d3ae17fde04d3865d29c259c/src/userbase-js/ws.js#L554))

Reply:
Added!